gitclone/gitclone.go

```diff
@@ -35,25 +37,28 @@ type Config struct {
 	Branch                string
 	PRDestBranch          string
 	PRSourceRepositoryURL string
-	PRMergeBranch         string
+	PRMergeRef            string
+	PRUnverifiedMergeRef  string
 	PRHeadBranch          string
 
 	ResetRepository bool
-	BuildURL        string
-	BuildAPIToken   string
```

Comment:
These are no longer needed here in the `Config` struct because we have instances of `bitriseapi.PatchSource` and `bitriseapi.MergeRefChecker` that contain these params.